dev3/2024-07-02/example-metadata/min-specimen-biosample.json

```diff
@@ -0,0 +1,74 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "acquisiton_method": {
+                "@reverse": "https://schema.org/result",
+                "@type": "@id"
+            },
+            "biological_entity": "https://schema.org/about",
+            "preparation_method": "https://www.wikidata.org/wiki/Property:P1537"
+        }
+    ],
+    "@graph": [
+        {
+            "@type": "CreativeWork",
+            "@id": "ro-crate-metadata.json",
+            "conformsTo": {
+                "@id": "https://w3id.org/ro/crate/1.1"
+            },
+            "about": {
+                "@id": "./"
+            }
+        },
+        {
+            "@id": "./",
+            "@type": "Dataset",
+            "name": "Light microscopy photo of a fly",
+            "description": "Light microscopy photo of a fruit fly.",
+            "licence": "https://creativecommons.org/licenses/by/4.0/",
+            "hasPart": {
+                "@id": "./dros-mel-image.zarr/"
+            }
+        },
+        {
+            "@id": "./dros-mel-image.zarr/",
+            "@type": "Dataset",
+            "name": "OME-ZARR files",
+            "description": "the ome zarr files of the fly.",
+            "acquisition_method": [
+                "_:b0"
+            ],
+            "preparation_method": [
```

Comment:
Another synatic point: I imagine the wider audience for these would prefer embedded blank nodes rather than references.

Reply:
While i 100% agree and much prefer embedded (not sure what to call non-flattened), compacted json-ld, the RO-Crate standard asks for flattened:

```The RO-Crate Metadata File MUST contain RO-Crate JSON-LD; a valid [JSON-LD 1.0](https://www.w3.org/TR/2014/REC-json-ld-20140116/) document in [flattened](https://www.w3.org/TR/json-ld/#flattened-document-form) and [compacted](https://www.w3.org/TR/json-ld/#compacted-document-form) form``` 

from: https://www.researchobject.org/ro-crate/specification/1.1/structure.html

Maybe the flattened structure less of a hard requirement than it being JSON-LD (JSON-LD libraries shouldn't have issues converting between the different profiles) but i went with flattened being a hard requirement just in case. Would be worth investigating further.

Reply:
> RO-Crate Metadata File MUST contain RO-Crate JSON-LD; a valid [JSON-LD 1.0](https://www.w3.org/TR/2014/REC-json-ld-20140116/) document in [flattened](https://www.w3.org/TR/json-ld/#flattened-document-form) and [compacted]

Wow! 🫨 TIL. I will ask. I find that quite surprising.

Reply:
The flattened structure is a hard requirement.

This part of the spec appendix has more formal extra guidance and clarity on this
https://www.researchobject.org/ro-crate/specification/1.1/appendix/jsonld#describing-entities-in-json-ld

And the 1.2-DRAFT version is clearer on when to use local identifiers vs blank nodes https://www.researchobject.org/ro-crate/specification/1.2-DRAFT/appendix/jsonld#describing-entities-in-json-ld

In particular, 1.2-DRAFT states:
> The use of a blank node identifier SHOULD be taken as hint by RO-Crate presentation software* to display the entity in-line, not as a separate entity with its own view, such as a page.

* e.g. Crate-O or roc2html, but I don't know if they are implemented in this way